### PR TITLE
(VDB-1228) Median kiss + diss transformers

### PR DESCRIPTION
**Pull request by gslaughl**

### CODE REVIEW

File: test_config/test_config.go
Comment (Gslaughl) [inline test_config/test_config.go]: I had to make `CleanTestDB` stop panicking on error in order to get the tests consistently passing in Travis. I can't figure out exactly why it keeps erroring out, nor can I reproduce the error locally. So, I'll need to dig into it a little more before merging, but planning to hold off on that since these transformers aren't top-priority right now.

File: transformers/events/median_diss/single/single_suite_test.go
Comment (yaoandrew) [inline transformers/events/median_diss/single/single_suite_test.go]: Updated function name needed here.

File: db/migrations/20200203102132_create_bid_event_table.sql
Comment (Gslaughl) [inline db/migrations/20200203102132_create_bid_event_table.sql]: This line fixes the flaky test we've been seeing, by specifying that the `CASCADE DELETE` should happen at the end of the transaction. This way we won't run into the issue where a trigger is trying to update a row that was already deleted, because the row won't have been deleted yet.

[Here](https://www.postgresql.org/docs/9.1/sql-set-constraints.html) are some docs about deferrable constraints if you're interested

Comment (rmulhol) [reply db/migrations/20200203102132_create_bid_event_table.sql:4]: Don't think we want to modify the earlier migration here. Is it possible to update this column in a new migration?

File: db/migrations/20200310105940_create_median_kiss_single.sql
Comment (rmulhol) [inline db/migrations/20200310105940_create_median_kiss_single.sql:26]: Not an issue right now but _if_ this is merged after the backfill stuff then we'll need updated timestamps on here to make sure they come after that. Mostly just posting as a general reminder to make sure all new migrations are the latest against staging for every PR

File: db/migrations/20200203102132_create_bid_event_table.sql
Comment (Gslaughl) [reply db/migrations/20200203102132_create_bid_event_table.sql:4]: Haha oh yeah 🤦‍♂ 

Can def put this into a separate migration

File: transformers/integration_tests/median_diss_single_test.go
Comment (rmulhol) [inline transformers/integration_tests/median_diss_single_test.go:18]: obsolete?

**Review by rmulhol — APPROVED** [review]
LGTM! :shipit:

File: transformers/integration_tests/median_diss_single_test.go
Comment (Gslaughl) [reply transformers/integration_tests/median_diss_single_test.go:18]: Looking in etherscan I still can't find any diss events, unfortunately

**Pull request merged**
